components/camera/camera_test.go

```diff
@@ -104,12 +106,16 @@ func TestNewCamera(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, props.SupportsPCD, test.ShouldBeFalse)
 	test.That(t, props.IntrinsicParams, test.ShouldBeNil)
+	test.That(t, props.FrameRate, test.ShouldEqual, 0.0) // test frame rate when it is not set
+
```

Comment:
@seanavery do we care about a zero frame rate when it is not set?

Reply:
I think this is fine that `FrameRate` in the`Properties` struct will default to 0 if not defined. 

The API `GetPropertiesResponse` will default to nil if not set as seen below.